functions.php

```diff
@@ -45,3 +46,46 @@ function pullData(PDO $db):array{
 $players = $query->fetchAll();
 return $players;
 }
+
+
+
+
+/**
+ * This function prepares the stats for entry into the databse by binding them to params and making a connection to the db
+ *
+ * @param string $name
+ * @param int $points
+ * @param int $games
+ * @param int $rings
+ * @param PDO $db
+ */
+function insertData ($name,$points,$games,$rings,$db){
+    $dbInsert = $db->prepare("INSERT INTO `nba` (`name`, `points`, `games`, `rings`) VALUES (:player, :points, :games, :rings)");
+    if (strlen($name <= 30)){
+        $dbInsert->bindParam(':player', $name);
+    } else {
+        header('input.php');
+    }
+    $dbInsert->bindParam(':points', $points);
+    $dbInsert->bindParam(':games', $games);
+    $dbInsert->bindParam(':rings', $rings);
+    $dbInsert->execute();
+    header('location: display.php');
```

Comment:
I would suggest that you don't do the redirects in this function.

It's not that you should never redirect from a function, but in this case if you instead made the function return a boolean value, it would give whoever is calling your function the choice of what to do next.